src/components/DownloadContactListButton.tsx

```diff
@@ -14,15 +14,8 @@ export function DownloadContactListButton (props: DownloadButtonProps) {
     const uniqueProperties = getUniqueListBy(arrayOfPropertyObjects, 'id')
     const csvRows = uniqueProperties.map((result: UnprocessedResultsFromCRM) => {
         if (result.owner_details && Array.isArray(result.owner_details)) {
-            let mobile
-            let workPhone
-            if (!result.owner_details[0]) {
-                mobile = result.owner_details[1].Mobile
-                workPhone = result.owner_details[1].Work_Phone
-            } else {
-                mobile = result.owner_details[0].Mobile
-                workPhone = result.owner_details[0].Work_Phone
-            }
+            const mobile = result.owner_details.find((owner: OwnerType) => owner.Mobile)
```

Comment:
.find will return the owner record rather than the mobile field.

Maybe 
```
const mobileNumbers = result.owner_details.map((owner: OwnerType) => owner.Mobile).filter((mobile: string) => mobile);
const mobile = mobileNumbers.length > 0 ? mobileNumbers[0] : null;
```